etc/proto/run.sh

```diff
@@ -55,6 +56,12 @@ protoc \
     --jsonschema_opt="json_fieldnames" \
     --doc_opt="json,proto-docs.json" \
     --doc2_opt="markdown,proto-docs.md" \
+    --grpc-gateway_out v2/src \
+    --grpc-gateway_opt logtostderr=true \
+    --grpc-gateway_opt paths=source_relative \
+    --grpc-gateway_opt generate_unbound_methods=true \
+    --openapiv2_out v2/src/openapi \
```

Comment:
It seems like this is not generating any methods; it's only generating types.

Reply:
Ah, this also needs "generate_unbound_methods": https://github.com/grpc-ecosystem/grpc-gateway/blob/main/protoc-gen-openapiv2/main.go#L40C18-L40C18

Reply:
Added options for unbound methods and single swagger.json file

Reply:
Oh, good idea!